enterprise/migrations/0101_move_data_to_saved_for_later.py

```diff
@@ -0,0 +1,30 @@
+# Generated by Django 2.2.13 on 2020-06-30 20:22
+
+from django.db import migrations
+
+
+# saved_for_later will replace the marked_done field
+def move_data_from_marked_done_to_saved_for_later(apps, schema_editor):
+    EnterpriseCourseEnrollment = apps.get_model('enterprise', 'EnterpriseCourseEnrollment')
+    course_enrollments = EnterpriseCourseEnrollment.objects.all()
+    for course_enrollment in course_enrollments:
+        course_enrollment.saved_for_later = course_enrollment.marked_done
+    EnterpriseCourseEnrollment.objects.bulk_update(course_enrollments, ['saved_for_later'])
+
+
+def move_data_from_saved_for_later_to_marked_done(apps, schema_editor):
+    EnterpriseCourseEnrollment = apps.get_model('enterprise', 'EnterpriseCourseEnrollment')
+    course_enrollments = EnterpriseCourseEnrollment.objects.all()
+    for course_enrollment in EnterpriseCourseEnrollment.objects.all():
```

Comment:
```python
for course_enrollment in course_enrollments
```
so you don't call the same thing twice?

Reply:
yeah, good call, I did it in the function above.